lm_eval/tasks/__init__.py

```diff
@@ -28,7 +28,10 @@ def __init__(self,
 
         self.task_group_map = collections.defaultdict(list)
 
-        self.task_config = utils.simple_parse_args_string(task_config)
+        if task_config is not None:
+            self.task_config = utils.simple_parse_args_string(task_config)
+        else:
```

Comment:
may I ask when task_config can be None? just curious

Reply:
When I didn't give task_config in command line argument, it occurs error 😢 .

Reply:
oh didn't check that. thank you for modifying it 👍